content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,232 @@
+
+Click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for the
+incoming returned product(s). A :guilabel:`Return` smart button will then be added to the top of
+the ticket.
+
+.. image:: after_sales/after-sales-return-smart-button.png
+   :align: center
+   :alt: View of the return smart button on a helpdesk ticket.
+
+.. seealso::
+   :doc:`/applications/sales/sales/products_prices/returns`
+
+Send products for repair from a ticket
+======================================
 
-Grant coupons from tickets
---------------------------
+If the ticket is related to an issue with a faulty or broken product, a repair order can be created
+from the *Helpdesk* ticket, and managed through the *Repair* application.
 
-First, be sure to have your *Coupon Program* planned in the *Sales* or *Website* application. Then,
-in *Helpdesk*, open your ticket, click on *Coupon*, and choose the respective one.
+To create a new repair order, open a :menuselection:`Helpdesk` ticket and click on the
+:guilabel:`Repair` button in the upper left corner.
 
-.. image:: after_sales/aftersales4.png
+Clicking the :guilabel:`Repair` button opens a blank :guilabel:`Repair Reference` form.
+
+.. image:: after_sales/after-sales-repair-reference.png
    :align: center
+   :alt: View of a repair reference page.
+
+.. note::
+   If a product was specified in the :guilabel:`Product` field on the ticket, it will be added to
+   the :guilabel:`Product to Repair` field automatically. If not, click into the field to select a
+   product from the drop down.
+
+Fill out the :guilabel:`Repair Description` field with a brief explanation of the issue. Click the
+:guilabel:`Sale Order` field and select the :abbr:`SO (Sales Order)` from which the product that is
+being repaired originated from. If a *Return* has been initiated for the product, select it from the
+drop-down in the :guilabel:`Return` field.
 
-Repairs from tickets
---------------------
+Choose an :guilabel:`Invoice Method` from the drop-down. Select :guilabel:`Before Repair` or
+:guilabel:`After Repair` to generate an invoice before or after the work is completed. Selecting
+:guilabel:`No Invoice` means that an invoice cannot be generated for this service.
 
-Clicking on *Repair* option, on your ticket page, a new repair order form is shown. Fill in the
-information as needed and choose the next step.
+If parts are required for the repair, they can be added in the :guilabel:`Parts` tab. Services can
+be added as product lines on the :guilabel:`Operations` tab. Additional information for the internal
+repair team can be added to the :guilabel:`Repair Notes` tab. Information for the customer can be be
+added to the :guilabel:`Quotation Notes` tab, and will be automatically added to the PDF of the
+quotations generated from this :guilabel:`Repair Reference`.
 
-.. image:: after_sales/aftersales5.png
+A :guilabel:`Repairs` smart button will be added to the ticket, linking to the repair order.
```

Comment:
There's a break in the logic of how and why the smart button is added because of the supplemental block that precedes this sentence, so I would reiterate quickly what we just did, or what prompts the smart button to appear.